diff --git a/node_modules/@types/pdfkit/index.d.ts b/node_modules/@types/pdfkit/index.d.ts
index b878887..3cfce32 100755
--- a/node_modules/@types/pdfkit/index.d.ts
+++ b/node_modules/@types/pdfkit/index.d.ts
@@ -436,6 +436,7 @@ declare namespace PDFKit {
         output(fn: any): void;
         end(): void;
         toString(): string;
+        addNamedDestination(name: string): void;
     }
 }
 
